bots/images/scripts/continuous-atomic.setup

```diff
@@ -47,6 +47,11 @@ echo "kernel.core_pattern=|/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e"
 ostree remote add --set=gpg-verify=false centos-atomic-continuous https://ci.centos.org/artifacts/sig-atomic/rdgo/centos-continuous/ostree/repo/
 rpm-ostree rebase centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
 
+# Download the libssh RPM which we'll use for package ovelay
+# The only way to do this is via a container
+. /etc/os-release
+docker run --rm --volume=/etc/yum.repos.d:/etc/yum.repos.d:z --volume=/root/rpms:/tmp/rpms:rw,z "$ID:$VERSION_ID" /bin/sh -cex "yum install -y findutils && yum install -y --downloadonly libssh && find /var -name '*.rpm' | while read rpm; do mv -v \$rpm /tmp/rpms; done"
```

Comment:
Learned something new today -- I didn't expect `sh -cex "cmd"` to work, as "cmd" is the argument of `-c`, not of `-x`. But it does!